.gitignore

```diff
@@ -15,6 +15,9 @@ playwright-report/
 .DS_Store
 *.pem
 
+# Tarballs
+*.tgz
```

Comment:
I added this for testing but I think it is fine for this to be added. If anything it adds protection.